nautobot/dcim/filters.py

```diff
@@ -882,6 +882,16 @@ class InterfaceFilterSet(
         method="filter_kind",
         label="Kind of interface",
     )
+    parent_interface_id = django_filters.ModelMultipleChoiceFilter(
+        field_name="parent_interface",
+        queryset=Interface.objects.all(),
+        label="Parent interface (ID)",
+    )
+    bridge_id = django_filters.ModelMultipleChoiceFilter(
+        field_name="bridge",
+        queryset=Interface.objects.all(),
```

Comment:
Should we limit these query sets to only allow selecting Interfaces with the type of `TYPE_BRIDGE`? Same with Lags below?

Reply:
Same would apply for VM Interfaces as well

Reply:
VMInterface has no type field